File: app/src/main/java/com/wugx_utils/test/TestContract.java
package com.wugx_utils.test;

import com.wugx_autils.mvp.view.BaseView;

/**
 * @author Wugx
 * @date 2018/11/8
 */
public interface TestContract {
    interface TestView extends BaseView {
        void showDatas(String result);
    }
}

File: kotlin_autils/src/main/java/com/wugx/kotlin_autils/TestContract.kt
package com.wugx.kotlin_autils

import com.wugx_autils.mvp.view.BaseView

/**
 *
 *
 *@author Wugx
 *@date   2018/12/18
 */
interface TestContract {
    interface IView : BaseView {
        fun showDatas(content: String)
    }
}